tensorflow/python/framework/test_util.py

```diff
@@ -1382,6 +1382,8 @@ def decorated(self, *args, **kwargs):
   return decorator
 
 
+@deprecation.deprecated(
+    None, "Use `tf.config.experimental.list_physical_devices('GPU')` instead.")
```

Comment:
Are we going to try to deprecate this in 2.0?
Do you think if we should just rewrite this to use `tf.config.experimental.list_physical_devices`
As this will be a pretty substantial change and I do not think we will be able to edit everything in time.

Reply:
The behavior is subtly different so I can't apply a re-write rule.

On second thought I'll close this PR and first make the new API not experimental in time for 2.1.